src/webgpu/util/floating_point.ts

```diff
@@ -39,7 +40,7 @@ import {
 } from './math.js';
 
 /** Indicate the kind of WGSL floating point numbers being operated on */
-export type FPKind = 'f32' | 'abstract';
+export type FPKind = 'f32' | 'abstract' | 'f16';
```

Comment:
übernit: I would have put `f16` next to `f32`.

Reply:
Done